packages/graphiql/resources/renderExample.js

```diff
@@ -48,6 +48,14 @@ function onTabChange(tabsState) {
   updateURL();
 }
 
+async function confirmCloseTab(_index) {
+  // eslint-disable-next-line no-alert
+  if (window.confirm('Are you sure you want to close this tab?')) {
+    return true;
+  }
+  return false;
+}
```

Comment:
this now adds confirmation on netlify preview, so it will be annoying
<img width="1649" alt="image" src="https://github.com/user-attachments/assets/f20ec3b2-185f-44d2-815a-86fb01f08b84">